contracts/contracts/src/functioncall/interface/CrosschainLockingInterface.sol

```diff
@@ -42,5 +42,4 @@ interface CrosschainLockingInterface {
      */
     function getActiveCallCrosschainRootTxId() external view returns (bytes32);
 
-    function isSingleBlockchainCall() external view returns (bool);
```

Comment:
This interface is used by contracts that implement lockable storage. Lockable storage components need to know if a call is a single blockchain call or a crosschain call.